docs/02-deployment-guide.md

```diff
@@ -176,20 +192,23 @@ if you run the Docker container multiple times you will still have details of th
 If you add a command on the end, you can run a specific script like this:
 
 ```shell
-sleeper deployment test/deployAll/deployTest.sh myinstanceid myvpc mysubnet
+sleeper builder sleeper/scripts/test/deployAll/deployTest.sh myinstanceid myvpc mysubnet
 ```
 
+There is also a [Nix shell](https://nixos.org/) file at the root of the repository, which can give you a shell with all dependencies required
+to build, deploy, and interact with instances.
+To enter the Nix shell you can run the following command
```

Comment:
This is an incomplete sentence, did you mean to remove this paragraph after referring to the dev guide?